### PR TITLE
Update Da Spec

### PR DESCRIPTION
**Pull request by preston-evans98**

### CODE REVIEW

File: sdk/specs/interfaces/da.md [hunk 1]
```diff
@@ -74,124 +74,197 @@ bundles containing unpopular transactions. So, the L1 needs to be censorship res
...
+A proof showing that each item in an associated vector is included in some state commitment. For example,
+this could be a list of merkle siblings.
 
- | Name        | Type        | Description                                 |
- |-------------|-------------|---------------------------------------------|
- | `blockhash` | `Blockhash` | The hash of the DA layer block to be parsed |
+### Type: `CompletenessProof`
 
-* **Response:**
+A proof showing that each an associated vector does not omit any "relevant" transactions. For example, this could be a
```
Comment (cemozerr) [inline sdk/specs/interfaces/da.md:105]:
```
.. each an associated vector ...
```

This looks wrong. Was there a typo?

File: sdk/specs/interfaces/da.md [hunk 2]
```diff
@@ -74,124 +74,197 @@ bundles containing unpopular transactions. So, the L1 needs to be censorship res
...
+merkle proof of the items immediately preceding and following a particular Celestia namespace. This type may be
+the unit struct if no completeness proof is required.
 
- | Name  | Type                        | Description                                                  |
- |-------|-----------------------------|--------------------------------------------------------------|
- | `txs` | `iterable<BlobTransaction>` | A list of L1 transactions ("data blobs"), with their senders |
+### Type: `Blockheader`
 
-### `get_relevant_txs_with_proof`
+Must include a `prev_hash` field. Must provide a function to compute its canonical hash
```
Comment (cemozerr) [inline sdk/specs/interfaces/da.md:111]: nit:
```suggestion
Must include a `prev_hash` field. Must provide a function to compute its canonical hash.
```

**Review by cemozerr — APPROVED** [review]
Made a few small comments, but generally LGTM.

File: sdk/specs/interfaces/da.md (hunk 1 above)
Comment (preston-evans98) [reply sdk/specs/interfaces/da.md]: Yep, the each is a type. Removed!

**Review by cemozerr — APPROVED** [review]
Made a few small comments, but generally LGTM.

**Pull request merged**